src/pipeline.h

```diff
@@ -159,6 +161,8 @@ struct PipelineBaton {
     thresholdGrayscale(true),
     trimTolerance(0),
     gamma(0.0),
+    linearA(1.0),
```

Comment:
Just spotted that these fields need to be initialised in the same order as they are declared to avoid a compiler warning.
```
../src/pipeline.h:83:10: warning: ‘PipelineBaton::gamma’ will be initialized after [-Wreorder]
   double gamma;
          ^
../src/pipeline.h:81:10: warning:   ‘double PipelineBaton::linearA’ [-Wreorder]
   double linearA;
          ^
```

Reply:
I like to change this - but how do i actually see these compiler warning. I use npm install to build a new release (unfortunately under windows) and there i can't see these warnings. 
(I have to admit that I'm not programming cpp for so long.)

Reply:
Thanks for updating - MSVC doesn't appear to warn about this but gcc and clang do.

Reply:
https://visualstudio.uservoice.com/forums/121579-visual-studio-ide/suggestions/2553854-c-compiler-should-warn-about-wrong-member-initia